<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GraphHomography</class>
 <widget class="QWidget" name="GraphHomography">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>890</width>
    <height>570</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>18</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Графики</string>
  </property>
  <widget class="QPushButton" name="pushButtonFileOpen">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>120</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Открыть</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelFilterSize">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>10</y>
     <width>90</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Размер</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBoxFilterSize">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>10</y>
     <width>60</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimum">
    <number>3</number>
   </property>
   <property name="maximum">
    <number>20</number>
   </property>
   <property name="value">
    <number>3</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBoxFilterLimit">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>10</y>
     <width>70</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimum">
    <number>0</number>
   </property>
   <property name="maximum">
    <number>255</number>
   </property>
   <property name="singleStep">
    <number>10</number>
   </property>
   <property name="value">
    <number>100</number>
   </property>
  </widget>
  <widget class="QLabel" name="labelFilterLimit">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>10</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Порог</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditFileName">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>10</y>
     <width>300</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: white;</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBoxFilter">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>10</y>
     <width>120</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Фильтр</string>
   </property>
  </widget>
  <widget class="QCustomPlot" name="customPlotAngle">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>430</width>
     <height>250</height>
    </rect>
   </property>
  </widget>
  <widget class="QCustomPlot" name="customPlotScale">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>310</y>
     <width>430</width>
     <height>250</height>
    </rect>
   </property>
  </widget>
  <widget class="QCustomPlot" name="customPlotDeltaX">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>50</y>
     <width>430</width>
     <height>250</height>
    </rect>
   </property>
  </widget>
  <widget class="QCustomPlot" name="customPlotDeltaY">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>310</y>
     <width>430</width>
     <height>250</height>
    </rect>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QFrame</extends>
   <header location="global">qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
